tests/qiskit_qir/test_qiskit_to_qir.py

```
@@ -0,0 +1,363 @@
+
+
+# ---------------------------------------------------------------------------
+# Error handling
+# ---------------------------------------------------------------------------
+
+
+class TestQiskitToQirErrors:
+    """Tests for error handling in qiskit_to_qir."""
+
+    def test_invalid_input_type(self):
+        with pytest.raises(TypeError, match="must be of type qiskit.QuantumCircuit"):
```

Ruff (0.15.13)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)